reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -71,6 +72,11 @@
  * @since 1.0.0
  */
 public abstract class PooledConnectionProvider<T extends Connection> implements ConnectionProvider {
+	/**
+	 * Context key used to propagate the caller event loop in the connection pool subscription.
+	 */
+	final static String CONTEXT_CALLER_EVENTLOOP = " callereventloop";
```

Comment:
Do we need this space at the beginning of the string?

```suggestion
	final static String CONTEXT_CALLER_EVENTLOOP = "callereventloop";
```